src/ai/backend/manager/models/image.py

```diff
@@ -161,6 +155,7 @@ class ImageRow(Base):
     )
     tag = sa.Column("tag", sa.TEXT)
     registry = sa.Column("registry", sa.String, nullable=False, index=True)
+    registry_id = sa.Column("registry_id", sa.String, nullable=True)
```

Comment:
```suggestion
    registry_id = sa.Column("registry_id", GUID, nullable=True)
```
Let's define this column as customized UUID type